nimbus-features/messagingFeature.yaml

```
@@ -56,6 +56,8 @@ features:
             AFTER_THREE_LAUNCHES_THIS_WEEK: app_cycle.foreground|eventSum('Weeks', 1, 0) >= 3
             USER_RECENTLY_INSTALLED:  days_since_install < 7
             USER_RECENTLY_UPDATED:    days_since_update < 7 && days_since_install != days_since_update
+            USER_INSTALLED_OR_UPDATED_YESTERDAY: days_since_update >= 1
+            APP_USED_SINCE_INSTALL_OR_UPDATE: app_cycle.foreground|eventSum('Days', days_since_update, 0) > 1
```

Nit: the event name needs quoting.
```suggestion
            APP_USED_SINCE_INSTALL_OR_UPDATE: "'app_cycle.foreground'|eventSum('Days', days_since_update, 0) > 1"
```